.gitpod.yml

```diff
@@ -19,6 +19,12 @@ tasks:
     command: source ./install-terraform.sh
   - name: Install Visual Studio Code extensions
     command: source ./install-vsextensions.sh
+  - name: Install PostgreSQL
+    command: source ./install-postgres.sh
+  - name: Install Google Cloud SDK
+    command: |
+      source ./install-gcloud.sh
+      gp sync-done gcloud-setup
```

Comment:
**Thought:** I'm not sure this will do anything without an `gp sync-await`. This will fire an event, but since there are no listener commands, it won't really do anything. Doesn't hurt to keep, but also doesn't add anything. 

The tasks will run in parallel, so there's only a need for this if the gcloud installation relies on another command. 

🙏 

Source: https://www.gitpod.io/docs/references/gitpod-cli#sync-await